cdc/kv/regionlock/region_range_lock.go

```diff
@@ -150,12 +279,12 @@ type RegionRangeLock struct {
 	totalSpan         tablepb.Span
 	changefeedLogInfo string
 
-	mu                sync.Mutex
-	rangeCheckpointTs *rangeTsMap
-	rangeLock         *btree.BTreeG[*rangeLockEntry]
-	regionIDLock      map[uint64]*rangeLockEntry
-	stopped           bool
-	refCount          uint64
+	mu              sync.RWMutex
+	rangeResolvedTs *rangeTsMap
+	rangeLock       *btree.BTreeG[*rangeLockEntry]
```

Comment:
The rangeResolvedTs only maintain information about regions that are not locked.  Also, it appears that there is no overlap between the data maintained by rangeResolvedTs and rangeLock.

Is it possible to use single btree data to maintain locked and unlocked regions?

Reply:
Good idea, we can take a try later, in another PR.